tests/Resources/Symfony/Controller/FormErrorController.php

```diff
@@ -8,26 +8,28 @@
 use Symfony\Component\Form\Extension\Core\Type\FormType;
 use Symfony\Component\Form\FormConfigInterface;
 use Symfony\Component\Form\FormError;
-use Symfony\Component\Form\FormErrorIterator;
 use Symfony\Component\Form\FormInterface;
+use Traversable;
 
 class FormErrorController extends AbstractController
 {
     /**
      * @return array{
      *     name: string,
-     *     errors: FormErrorIterator<FormError>,
+     *     errors: FormError[],
      *     config: FormConfigInterface<null>,
      *     all: FormInterface<null>[]
      * }
      */
     public function __invoke(): array
     {
         $form = $this->createForm(FormType::class);
+        /** @var Traversable<FormError> $errors */ // @phpstan-ignore varTag.nativeType
+        $errors = $form->getErrors();
```

Comment:
Very hard time to get phpstan accept the type. Tried about a dozen variantions which work locally, but failed here. Final solution was to just ignore it. Mostlikely in future phpstan version this ignore can be removed again.